src/WorkItemMigrator/Migration.Common/MigrationContext.cs

```diff
@@ -9,8 +10,8 @@ namespace Migration.Common
     public class MigrationContext
     {
         public static MigrationContext Instance { get; private set; }
-        public string AttachmentsPath { get { return Path.Combine(MigrationWorkspace, "Attachments"); } }
-        public string UserMappingPath { get { return Path.Combine(MigrationWorkspace, "users.txt"); } }
+        public string AttachmentsPath { get; private set; }
```

Comment:
arent you making these as mandator configs ?

Reply:
As we discussed. Attachment path is a already mandatory config property, see: https://github.com/solidify/jira-azuredevops-migrator/blob/master/docs/config.md#properties. These changes are consistent with the docs.